public/beacon/catcher.php

```diff
@@ -7,11 +8,13 @@
 
 // Hacking quickly to handle Cross Origin Requests
 // Better if we implement this in NIGIX level
+// $origin = !empty($_SERVER['HTTP_ORIGIN']) ? $_SERVER['HTTP_ORIGIN'] : '*';
 $origin = !empty($_SERVER['HTTP_ORIGIN']) ? $_SERVER['HTTP_ORIGIN'] : '*';
-$originHeader = 'Access-Control-Allow-Origin: ' . $origin;
+$originHeader = 'Access-Control-Allow-Origin: '.$origin;
 
-if ($_SERVER['REQUEST_METHOD'] == 'OPTIONS') {
+if ('OPTIONS' == $_SERVER['REQUEST_METHOD']) {
     header($originHeader);
+    header('Access-Control-Allow-Credentials: true');
```

Comment:
Hello @korzol 

I think that this wasn't suppose to be pushed to the repo header('Access-Control-Allow-Credentials: true') 

What is the idea to have "Access-Control-Allow-Credentials: true" ?